nzedb/Desura.php

```diff
@@ -254,9 +252,8 @@ public function getAll()
 	 */
 	private function getUrl($fetchURL, $usePost = false)
 	{
-		if (isset($fetchURL)) {
-			$this->_ch = curl_init($fetchURL);
-		}
+		$this->_ch = curl_init($fetchURL);
```

Comment:
When changing lines with php functions, check if it is optimal to namespace them. e.g. this should be:
$this->_ch = \curl_init($fetchURL);